LEGO1/mxentity.h

```diff
@@ -24,6 +24,8 @@ class MxEntity : public MxCore
   {
     return !strcmp(name, MxEntity::ClassName()) || MxCore::IsA(name);
   }
+
+  virtual MxResult MxEntity::SetEntityId(MxS32 p_id, MxAtomId *p_atom);
```

Comment:
Including class name in definition is not standards compliant. Also if the above change is used, exchange `MxAtomId *` for `const MxAtomId &`.